test/integration/node-specific/feature_flags.test.ts

```diff
@@ -42,4 +45,87 @@ describe('Feature Flags', () => {
       });
     }
   });
+
+  describe('@@mdb.enableMongoLogger', () => {
+    let cachedEnv;
+    const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+
+    before(() => {
+      cachedEnv = process.env;
+    });
+
+    after(() => {
+      process.env = cachedEnv;
+    });
+
+    context('when enabled', () => {
+      context('when logging is enabled for any component', () => {
+        before(() => {
+          process.env.MONGODB_LOG_COMMAND = SeverityLevel.EMERGENCY;
+        });
+
+        it('enables logging for MONGODB_LOG_COMMAND', () => {
+          const client = new MongoClient('mongodb://localhost:27017', {
+            [loggerFeatureFlag]: true
+          });
+          expect(client.mongoLogger.componentSeverities).to.have.property(
+            'command',
+            SeverityLevel.EMERGENCY
+          );
+        });
+      });
+
+      context('when logging is not enabled for any component', () => {
+        before(() => {
+          process.env = {};
+        });
+
+        it('does not enable logging', () => {
+          const client = new MongoClient('mongodb://localhost:27017', {
+            [loggerFeatureFlag]: true
+          });
+          expect(client.mongoLogger.componentSeverities).to.have.property(
```

Comment:
we may as well add expectations for the rest of the available components (`for..of` over the possible components + default would work)